src/main/java/at/hannibal2/skyhanni/features/fishing/tracker/FishingProfitTracker.kt

```diff
@@ -151,6 +151,7 @@ object FishingProfitTracker {
         }
 
         if (tracker.isInventoryOpen()) {
+            // TODO change
```

Comment:
More descriptive please, we've got a lot of these spread around and they're never gonna stop if we don't start adding more detail imo

Reply:
this is stuff i want to do in this pr before  getting it merged